resources/multisite_drupal_standard.make

```diff
@@ -289,7 +289,8 @@ projects[field_group][version] = "1.5"
 projects[field_group][patch][] = https://www.drupal.org/files/issues/field_group_label_translation_patch.patch
 
 projects[file_entity][subdir] = "contrib"
-projects[file_entity][version] = "2.2"
+projects[file_entity][version] = "2.4"
+projects[file_entity][patch][] = https://www.drupal.org/files/issues/D7-file_entity-file_description_missing-2893132-2.patch
```

Comment:
Can you add information of this patch?

 ; Issue #99999: blablablabla.
 ; https://www.drupal.org/node/99999
 ; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-999

Reply:
If you want to upgrade the module for the last version, maybe we can point directly the last commit.